prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md

```diff
@@ -0,0 +1,41 @@
+# prep-html-skeleton-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What tags are necessary for a complete HTML Skeleton?
+
+- What type of content belongs within the `<head>` of an HTML document?
+
+- What type of content belongs within the `<body>` of an HTML document?
+
+- Where must the `DOCTYPE` declaration appear in a valid HTML document?
+
```

Comment:
Don't forget to answer the quiz questions!

Reply:
<img width="1792" alt="Screenshot 2024-06-15 at 12 42 23 PM" src="https://github.com/devtanole/javascript-prep-0624-code-solutions/assets/161262712/4e7ad3b5-3344-4751-9c4e-844a26a3c3bd">